### PR TITLE
LPS-178773 | Added automation test FrontendDataSetViews#CanCreateDataSet

### PR DESCRIPTION
**Pull request by andreluis2022**
Description:
Add automation test about datasets

Plan:
- Given and When goto Control Panel>Datasets / Add new dataset, plus button
- And when filling the field name with the name
- And When you choose the Provider called channel
- And When clicking on the Save button
- And Then Confirm the name as expected
- And Then Confirm the Provider as expected

JIRA Ticket:
https://issues.liferay.com/browse/LPS-178773

### CODE REVIEW

**Comment by andreluis2022** [comment]
ci:test:sf

**Comment by andreluis2022** [comment]
ci:test:sf

**Comment by andreluis2022** [comment]
ci:test:sf

**Comment by andreluis2022** [comment]
ci:test:sf

**Comment by andreluis2022** [comment]
ci:test:sf

**Comment by andreluis2022** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/FrontendDataSetViews.testcase [hunk 1]
```diff
@@ -38,13 +38,38 @@ definition {
 
 	}
 
-	@description = "Confirm that the user can create a data set view"
-	@ignore = "Test Stub"
+	@description = "LPS-178773 - Confirm that the user can create a data set view"
 	@priority = 5
 	test CanCreateDataSet {
+		property custom.properties = "feature.flag.LPS-164563=true";
```
Comment (mateusralv) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/FrontendDataSetViews.testcase:44]: This property has already been added to the beginning of the file.

**Review by mateusralv — APPROVED** [review]

**Comment by andreluis2022** [comment]
ci:test:sf

**Comment by andreluis2022** [comment]
ci:test:frontend

**Comment by mateusralv** [comment]
ci:test:sf

**Comment by mateusralv** [comment]
ci:test:frontend

**Comment by andreluis2022** [comment]
ci:test:sf

**Comment by andreluis2022** [comment]
ci:test:sf

**Comment by mateusralv** [comment]
ci:test:frontend

**Comment by mateusralv** [comment]
ci:test:sf

**Comment by andreluis2022** [comment]
ci:test:sf

**Comment by andreluis2022** [comment]
ci:test:frontend

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/frontenddataset/FrontendDataSet.path [hunk 2]
```diff
@@ -139,13 +139,13 @@
...
+	<td>SELECT_TYPE_PROVIDER</td>
+	<td>//button[@class="dropdown-item"]//span[contains(.,'${key_type}')]</td>
 	<td></td>
 </tr>
 <tr>
-	<td>SELECT_TYPE_PROVIDER</td>
-	<td>//button[@class="dropdown-item"]//span[contains(.,'${key_type}')]</td>
+	<td>FILTER_OPTION</td>
+	<td>//div[@class='dropdown-menu show']//*[@class='dropdown-item' and contains(.,'${key_filter}')]</td>
 	<td></td>
```
Comment (mateusralv) [inline portal-web/test/functional/com/liferay/portalweb/paths/portlet/frontenddataset/FrontendDataSet.path:149]: Hey @andreluis2022 why were these changes made? I don't see them being used in your test 🤔

**Review by mateusralv — APPROVED** [review]
Just undo the changes on the paths that are not being used in the test. The test is ready to go. Good work @andreluis2022 🎉

**Comment by andreluis2022** [comment]
> Just undo the changes on the paths that are not being used in the test. The test is ready to go. Good work @andreluis2022 tada

Thanks @mateusralv

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/frontenddataset/FrontendDataSet.path (hunk 2 above)
Comment (andreluis2022) [reply portal-web/test/functional/com/liferay/portalweb/paths/portlet/frontenddataset/FrontendDataSet.path]: Done!!

**Comment by andreluis2022** [comment]
> Just undo the changes on the paths that are not being used in the test. The test is ready to go. Good work @andreluis2022 tada

Done!

**Comment by andreluis2022** [comment]
ci:test:sf

**Comment by mateusralv** [comment]
ci:forward

**Comment by andreluis2022** [comment]
ci:test:sf

**Comment by mateusralv** [comment]
ci:test:frontend

**Comment by andreluis2022** [comment]
> ci:test:frontend

Do I need anything else in this PR?

**Comment by mateusralv** [comment]
ci:forward

**Pull request closed without merging**